front/views/__snapshots__/AuthenticationView.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AuthenticationView Component view should render 1`] = `[Function]`;
```

Comment:
What is this ?, the same question for all three `.snap` files

Reply:
same as the above comment, thought it was still useful but indeed it is not so removed it, thanks